app/actions/remote/general.ts

```diff
@@ -50,6 +75,17 @@ export const doPing = async (serverUrl: string) => {
         return {error: {intl: pingError}};
     }
 
+    if (verifyPushProxy) {
+        let canReceiveNotifications = response?.data?.CanReceiveNotifications;
+
+        // Already verified or old server
+        if (deviceId == null || !canReceiveNotifications == null) {
```

Comment:
Following `getDeviceIdForPing`, it returns `undefined` if the status is `PUSH_PROXY_STATUS_VERIFIED`.

```suggestion
        if (deviceId == undefined || !canReceiveNotifications == null) {
```

Reply:
`== null` checks for undefined and null. I can do `=== undefined` to be more specific.